docs/pages/client_usage.mdx

```diff
@@ -0,0 +1,231 @@
+
+# Using The Pelican Client
+It is recommended that you start with our [Quick Start Guide](./user-quickstart-guide.mdx) where you can learn the basics of Pelican client commands and learn some useful terminology.
+
+## Before Starting
+
+### Assumptions
+
+Before using the Pelican client to interact with objects from your federation, this guide makes several assumptions:
+
+- You are on a computer where you have access to a terminal. The Pelican client is a command line tool.
+- You've already installed the version of Pelican appropriate for your system, and Pelican is accessible via your path. To test this on Linux, you can run
+```bash
+which pelican
+```
+which should output a path to the executable. If there is no output to this command, refer to the [Pelican installation docs](./install.mdx) to acquire a working installation.
+- You are roughly familiar with Pelican client commands and terminology as demonstrated with the [Quick Start Guide](./user-quickstart-guide.mdx)
+
+
+### Useful Terminology
+
+**Notes on Federations:**
+Below are some notes on federations on top of their terminology. Here is the example URL we used previously:
+```console
```

Comment:
Previously where? Not in this document. The quick start guide? If so, it should be something like "previously in the quick start guide" or something similar.